src/server/network/protocol/protocolgame.cpp

```diff
@@ -132,7 +132,10 @@ void ProtocolGame::AddItem(NetworkMessage &msg, const Item *item)
     	if (container && item->getWeaponType() == WEAPON_QUIVER && player->getThing(CONST_SLOT_RIGHT) == item) {
       		uint16_t ammoTotal = 0;
       		for (Item* listItem : container->getItemList()) {
-        		ammoTotal += listItem->getItemCount();
+				const ItemType &listType = Item::items[listItem->getID()];
+				if (listType.minReqLevel != 0 && player->getLevel() >= listType.minReqLevel) {
```

Comment:
same as above:

Do we need to check if listType.minReqLevel != 0?
player level is always > 0, I think the second condition is enough, isn't it?
also, I think if you define the minReqLevel for that arrow as 0 and there are only this type of arrows inside the quiver, "ammoTotal " won't be incremented even though player can use this ammo.